core/contextmenu.js

```diff
@@ -215,11 +215,11 @@ Blockly.ContextMenu.callbackFactory = function(block, xml) {
 Blockly.ContextMenu.blockDeleteOption = function(block) {
   // Option to delete this block but not blocks lower in the stack.
   // Count the number of blocks that are nested in this block.
-  var descendantCount = block.getDescendants(true).length;
```

Comment:
This boolean previously did nothing and is just a mistake.  Same below.  This change probably fixes a couple of compiler warnings.